File: python/one/rectangle.py
class Rectangle:
    def __init__(self, length, width):
        self.length = length
        self.width = width
    def get_area(self):
        return self.length * self.width
    def get_info(self):
        if self.length == self.width:
            return 'I am a square'
        else:
            return 'I am a rectangle'

File: java/one/Rectangle.java

public class Rectangle implements Shape{
	private double length;
	private double width;
	public Rectangle(double length, double width) {
		this.length = length;
		this.width = width;
	}
	
	public double getArea() {
		return length * width;
	}
	
	public String getInfo() {
		if(length == width) {
			return "I am a square with side length " + length;
		} else {
			return "I am a rectangle with length " + length + " and width " + width;
		}
	}
}
